_posts/en/2020-02-16-what-devops-looks-like.md

```diff
@@ -0,0 +1,121 @@
+- In order to increase the likelihood of success, departments have significantly reduced the scope of the projects and considerably extended their timelines. In the area of project governance, the Expenditure Management Information System and Global Case Management System have had important deliverables deferred without full analysis of the impacts and costs of not completing these projects. In three of the five projects examined, the project business cases did not identify measurable benefits or the benefits have not been measured. For example, the 2007 business case for the Secure Channel no longer quantifies the financial benefits.
+
+Once again these findings are a decade or more old and yet the issue pointed out are still relevant today. The underlying issue is slow response to change and a large lead time to be able to incorporate any change in requirements into projects and systems. One of the culprits of this is a large and complex release process which results in massive and risky releases.
+
+Here is a map of our current release process
+
+<div style = "display: block; width: inherit; height: 400px; text-align: center;">
+    <a href="/ITStrategy/assets/images/Feature-Intake-to-Release-BPM.png" style="display: block;">
+        <img src = "/ITStrategy/assets/images/Feature-Intake-to-Release-BPM.png"
+        alt = "Map of current release process including the different steps, handoffs and stakeholders"
+        style="width: 550px; height: 399px;"
+        />
+    </a>
+    <small> Our current release process ( You can click to expand this image )</small>
+</div>
+<br/>
+
+If you're looking at this for the first time it's almost impossible to comprehend. It is difficult for one person to completely describe all the stages that it involves. Our process is large, complex, involves many different stakeholders with distinct functional roles, manual checks and handoffs. Here are some statistics to help you understand the scale of the complexity.
+
+Assume you are a developer, you are tasked with adding 4 fields to a form on a front facing application, this is about two weeks of actual development work. You have to alter some front end code, add input validation in your views and perhaps alter a database schema to capture these fields. With our current process it will take you 3-5 months to release your changes to production. Within that time you will deal with
+
+- More than 13 stakeholders, both internal and external, with distinct functional roles as well as different intake and handoff processes. This includes but is not limited too:
```

Comment:
I'd say don't list them. Just mention there are 13 stakeholders with examples of their functions (e.g. analyst, developer, quality assurance, release process). This  so not to expose them as flaws.

Reply:
hmm, it's in the BPM diagram, the intention here was not to say that these stakeholders are good or bad but it is an objective fact all 13 are involved whenever there is a release

Reply:
"objective fact" and "should be published in the open" are not synonymous

I agree. Should abstract the list away.